Camera Window Buttons Component

leesuann opened the pull request:
- Live
- Gallery
- Record(No SdCard)
- Record/shoot Switch
- Under Up


![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/e806cdca-0695-4864-95b1-39c170717e16)
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/e217a7a0-16c7-46e3-8e0b-6a2f92e22630)
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/789a06f7-c101-454e-8767-42f6a76408a0)
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/df51ded3-bf4c-4daa-a0d8-312483bfc10e)
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/a13c1186-955f-49ca-abe1-815ccf8fc130)
![image](https://github.com/argosdyne/AELS_QGC/assets/92770122/b217835f-d121-4d76-8968-f6f8d76f4608)

leesuann committed:
Camera Window Buttons Component
- Live
- Gallery
- Record(No SdCard)
- Record/shoot Switch
- Under Up

leesuann committed:
_root to root

thanhkaist commented on custom/qgcresources.qrc:
```diff
@@ -52,6 +52,12 @@
+
+		<file alias="GalleryButton.svg"         >../resources/GalleryButton.svg</file>
+		<file alias="LiveButton.svg"            >../resources/LiveButton.svg</file>
+		<file alias="Shoot_Record_Switch.svg"   >../resources/Shoot_Record_Switch.svg</file>
```
Change file name to CamelCase

thanhkaist commented on custom/qgcresources.qrc:
```diff
@@ -52,6 +52,12 @@
+		<file alias="LiveButton.svg"            >../resources/LiveButton.svg</file>
+		<file alias="Shoot_Record_Switch.svg"   >../resources/Shoot_Record_Switch.svg</file>
+		<file alias="Record.svg"                >../resources/Record.svg</file>
+		<file alias="NoSDcard(Red).svg"         >../resources/NoSDcard(Red).svg</file>
```
Remove special characters ()

thanhkaist commented on src/QmlControls/CameraGalleryButton.qml:
```diff
@@ -0,0 +1,36 @@
+    id:     root
+    z : 3
+    property var rootWidth: null;
+    property var rootHeight: null;
```
not sure why you are using this property. For a custom item, you should set the implicitWidth and implicitHeight property instead

thanhkaist commented on src/QmlControls/CameraLiveButton.qml:
```diff
@@ -0,0 +1,33 @@
+    id:     root
+    z : 3
+    property var rootWidth: null;
+    property var rootHeight: null;
```
not sure why you are using this property. For a custom item, you should set the implicitWidth and implicitHeight property instead

thanhkaist commented on src/QmlControls/CameraMenuUnderUpButton.qml:
```diff
@@ -0,0 +1,33 @@
+Button {
+    id:     root
+    z : 3
+    property var rootWidth: null;
```
not sure why you are using this property. For a custom item, you should set the implicitWidth and implicitHeight property instead

thanhkaist commented on src/QmlControls/CameraRecordSwitchButton.qml:
```diff
@@ -0,0 +1,33 @@
+
+    Image {
+        anchors.fill: parent
+        source: "/res/Shoot_Record_Switch.svg"
```
The file name should be CamelCase

leesuann committed:
FileName Change to CamelCase
Change color to property
Change width and height to implictWidth and implictHeight

thanhkaist committed:
Merge branch 'Beta01.00.00' into AlesQGC_Camera_Buttons
